doc/changes/0.11.0.rst

```diff
@@ -95,6 +95,7 @@ Fixes
 
 Enhancements
 ------------
+- :bdg-dark:``Code`` Added ``screening_n_voxels`` parameter to :class:``~nilearn.decoding.Decoder`` and :class:``~nilearn.decoding.DecoderRegressor``
```

Comment:
this should be in latest.rst

also I am pretty sure the changes should affect the FREMClassifier and FREMRegressor too